README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Development
 
-Currently images from this repo need to be manually pushed by a developer. This can be done by running the following script after setting appropriate AWS credentials:
+?Currently images from this repo need to be manually pushed by a developer. This can be done by running the following script after setting appropriate AWS credentials:
```

Comment:
should there be a `?` here?

Reply:
well i had a question abt it https://github.com/safeinsights/setup-app/pull/24#discussion_r2012238139